tests/frontend/components/test_build_edit_section.py

```diff
@@ -20,8 +24,54 @@
         ),
     ],
 )
-def test_build_edit_section(build_edit_section):
-    title = build_edit_section.children[0].children
+def test_build_edit_section_return_correct_component(build_edit_section):
+    """Assert method returns html section."""
+    assert isinstance(build_edit_section, html.Section)
+
+
+@pytest.mark.parametrize(
+    ("build_edit_section"),
+    [
+        build_edit_section([], "", model.Variable(), ""),
```

Comment:
I must say I'm a bit uncertain about calling the function under test in the parametrize section. Typically the parametrize section will contain just data which make up the test cases and then the function under test will be called in the test itself.

I don't think it makes a major functional difference, it's more of a philosophical question.